static/js/views/create_cards/CardBuilderCost.js

```diff
@@ -14,18 +14,9 @@ export class CardBuilderCost extends CardBuilderBase {
     }
 
     cardInfo() {
-        return {
-            name: this.originalCardInfo.name, 
-            card_type: this.originalCardInfo.card_type, 
-            image: this.originalCardInfo.image, 
-            effects: this.originalCardInfo.effects, 
-            strength: this.originalCardInfo.strength, 
-            hit_points: this.originalCardInfo.hit_points, 
-            description: this.cardDescription(),
-            cost: this.cardCost(), 
-            power_points: this.powerPoints ? this.powerPoints : this.originalCardInfo.power_points,
-            author_username: this.originalCardInfo.author_username
-        };
+        let info = super.cardInfo();
```

Comment:
const

Reply:
done, though meandering thought is const not protecting object keys and array values is weird, dubious, and or smelly